src/Component/SelectMenu/StringSelectMenu.php

```diff
@@ -31,10 +31,7 @@ public function addOption(
             throw new TooManyOptionsException();
         }
 
-        $item = [
-            'label' => $label,
-            'value' => $value,
-        ];
+        $item = compact('label', 'value');
```

Comment:
I dislike using `compact`, feels like magic

Reply:
Understandable, having keys named exactly like the variable though is pretty much a waste of space. I'd say its more of a personal preference though, you can keep the arrays as-is here.